src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -445,9 +445,13 @@ public void run() {
                     try {
                         serverTask.execute();
                     } catch (RuntimeException e) {
-                        // If a runtime exception occurred in the server task, log and rethrow
-                        error("An error occurred while starting the server: " + e.getMessage(), e);
-                        throw e;
+                        // If devStop is true server was stopped with Ctl-c, do not throw exception
+                        AtomicBoolean devStop = getDevStop();
```

Comment:
The `getDevStop()` method shouldn't be necessary since `devStop` is an instance variable in DevUtil.

Reply:
Since the error in caught inside the `new Runnable() {` object we cannot access `this.devStop` directly.

Reply:
I don't see any compile error if I remove the line `AtomicBoolean devStop = getDevStop();`